### PR TITLE
Conan package identifier

### PR DESCRIPTION
**Pull request by adityanarayanp**
Package identifier that performs the file parsing logic for identification of dependency for CONAN packages.

### CODE REVIEW

**Comment by sumanthkb44** [comment]
PR looks fine ,we will merge this when we will plan for releasing the conan package clearing.

File: src/LCT.Common/Model/Config.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class Config
         public string[] JfrogNugetRepoList { get; set; }
         public string[] JfrogMavenRepoList { get; set; }
         public string[] JfrogPythonRepoList { get; set; }
+        public string[] JfrogConanRepoList { get; set; }
         public string[] DevDependentScopeList { get; set; }
```
Comment (karthika-g) [inline src/LCT.Common/Model/Config.cs:26]: Are there changes in appSettings.json file?If yes then push those as well

Comment (adityanarayanp) [reply src/LCT.Common/Model/Config.cs:26]: Yes, have updated the appSettings.json with the Conan configurations now.

File: CA.nuspec [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <package >
   <metadata>
     <id>continuous-clearing</id>
-    <version>4.0.0</version>
```
[the remark this answers is not in the document]
Comment (adityanarayanp) [reply CA.nuspec:7]: done

**Review by karthika-g — APPROVED** [review]
Approved

**Pull request merged**